crates/rattler_networking/Cargo.toml

```diff
@@ -38,6 +38,7 @@ task-local-extensions = { workspace = true }
 thiserror = { workspace = true }
 tracing = { workspace = true }
 url = { workspace = true }
+google-cloud-auth = "0.13.2"
```

Comment:
Could you add this as a workspace dependency? 

Google-cloud-auth also by default adds the default-tls features. Could you make the tls implementation configurable using the features of the crate?